webhook.rst

```diff
@@ -77,20 +77,27 @@ The Webhook component routing needs to be defined:
             ;
         };
 
-Currently, the following third-party mailer services support webhooks:
+Currently, the following third-party services support webhooks:
+
+======== ==========================================
+Service  Parser service name
+======== ==========================================
+Brevo    ``mailer.webhook.request_parser.brevo``
+Mailgun  ``mailer.webhook.request_parser.mailgun``
+Mailjet  ``mailer.webhook.request_parser.mailjet``
+Postmark ``mailer.webhook.request_parser.postmark``
+Sendgrid ``mailer.webhook.request_parser.sendgrid``
+Vonage   ``mailer.webhook.request_parser.vonage``
```

Comment:
This service (Vonage) can be removed from this list. It's not a `mailer` service, but one for `notifier`.

Reply:
I agree I have to update the service alias. But don't we want to keep all third party services supported by Webhook here?

Reply:
I made two separate lists in my other PR. One for Mailer and one for Notifier.

Reply:
We can also merge this PR as is, and then I'll fix it in the other PR. ^_^

Reply:
Suits me fine! Good for you @OskarStark? :slightly_smiling_face: